roles/network_plugin/fan/tasks/main.yml

```diff
@@ -0,0 +1,33 @@
+---
+
+- name: "Fan | Ensure ubuntu-fan installed"
+  apt:
+    name: "ubuntu-fan"
```

Comment:
i just tested it, i don't know if that's possible but a plugin network has to be supported by major linux distribs (e.g. Debian, RedHat). Hope there's a common way to install it...